### ISSUE TITLE
Show Less button overlaps and flies in the middle of the screen blocking content

### ISSUE THREAD

**Issue by sudheesh001** [adobe/adobe.github.com#51]
![adobe](https://cloud.githubusercontent.com/assets/4545925/11137953/ae5808ca-89e2-11e5-858e-d23ee5effcd4.png)

This is just a suggestion, Just the way the `Show More` button lies at the bottom of the list of repositories, clicking it should have a similar behavior where the `Show Less` button too should scroll down to the bottom of the list rather than hovering in the middle.

**Comment by erilyth** [comment]
@sudheesh001 But we wouldn't want to scroll down to the bottom of such a long list before we can press show less. It would be better if we added a little more padding to the page, and then put the show less button a little lower so it doesn't obscure the text.

**Comment by noraacl** [comment]
@sudheesh001 & @erilyth thanks for submitting this; will look into it.

**Comment by aishivam** [comment]
@sudheesh001  it does, but I don't think so it is an issue. We can easily scroll the content

**Comment by yugalmittal05** [comment]
i solved it.

**Comment by kushalv238** [comment]
@sudheesh001 I don't think it blocks the content and you'll have to scroll all the way to the bottom the collapse it, so a floating button is the perfect choice here. But the problem is that the button persists even if I leave the section, which does not look good.

This is the video representation:

https://github.com/adobe/adobe.github.com/assets/77831343/935ba2a3-2285-44dc-a5e4-6ff45e87bf50

I have solved this bug in my latest pr

**Comment by charan-mudiraj** [comment]
i'm unable to resolve the CLI Signing issue. i've already signed but still getting the issue. Please review my pull request.
